### ISSUE TITLE
Error using eslint-plugin-promise 5.1.0

### ISSUE THREAD

**Issue by MartinX3** [vuejs/eslint-config-standard#8]
```
$ npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! Found: eslint-plugin-promise@5.1.0
npm ERR! node_modules/eslint-plugin-promise
npm ERR!   dev eslint-plugin-promise@"5.1.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer eslint-plugin-promise@"^4.2.1" from @vue/eslint-config-standard@6.0.0
npm ERR! node_modules/@vue/eslint-config-standard
npm ERR!   dev @vue/eslint-config-standard@"6.0.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by xCykrix** [comment]
I believe this is due to `eslint-plugin-promise` changing their major version to `v5.1.0` (currently), and npm is just refusing to install it. You can change the version in your `package.json` for `eslint-plugin-promise` to `^4.3.0` to solve this issue temporarily until a fix is created. I encounter this doing maintenance on my packages :/

**Comment by sodatea** [comment]
Fixed in v6.1.0
